packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -661,15 +669,19 @@ class Viewport implements IViewport {
       RENDERING_DEFAULTS.MAXIMUM_RAY_DISTANCE
     );
 
+    if (this.flipHorizontal || this.flipVertical) {
+      this.flip({ flipHorizontal: false, flipVertical: false });
+    }
+
+    if (resetPan && resetZoom) {
```

Comment:
why it needs `resetZoom` in order to `resetPanOffset`?

Reply:
I still think resetCamera should neglect all the provided options and reset based on the FOV and actor bounds. IMO resetCamera should not go back to the state which you have defined with (`initialCenter` etc.). If you have put it in the `resetCamera` in order for the initial render to be correct, it can, instead, be a public API like `viewport.setPresentationState()` that accept those. 

CC'ing @swederik

Reply:
I'm even thinking that `PublicViewportInput` can ommit these presentational states, and the deverloper can just do 

`viewport.setStack(imageId).then(()=>{
  viewport.setPresentationState(initialRange, initialCenter, etc.) // we discussed these names should change
  viewport.render()
})